Sources/Persistence/DiskMemoryPersistenceStack.swift

```diff
@@ -74,11 +88,19 @@ public final class DiskMemoryPersistenceStack: PersistenceStack {
     private let fileManager: FileManager = FileManager.default
 
     private var usedDiskSize: UInt64 = 0 // Size in bytes
+    private var usedMemorySize: UInt64 = 0 // Size in bytes
 
-    public init(configuration: Configuration) {
+    private let performanceMetrics: PerformanceMetrics?
+
+    public init(configuration: Configuration,
+                performanceMetrics: PerformanceMetrics? = nil) {
```

Comment:
I would perhaps suggest injecting a configuration with the keys to use, to avoid these being essentially hardcoded to the constants. 
It also enables configuring other stuff in the future, maybe.

Reply:
I discussed this with @portellaa and we thought this could be an improvement done later when we have more information on different performance trackers. Is it worth doing now then?

Reply:
Fixed I think. Not sure it is what we had in mind but will commit soon.